pytest-selenium/tests/test_attribution.py

```diff
@@ -0,0 +1,63 @@
+
+import unittest
+from time import sleep
+
+import pytest
+from selenium import webdriver
+#from selenium.common.exceptions import StaleElementReferenceException
+
+class TestCitationtexturl():
+    #def setUp(self):
+        #self.browser = webdriver.Chrome()
+        #self.browser.implicitly_wait(3)
+    #def teardown(self):
+       # self.browser.quit()
+
+
+
```

Comment:
We need to add two more decorators here. We are currently using the pytest-testrail plugin which allows us to integrate with testrail. In order to let the plugin know we need to add a special marker to the test. 

`@markers.test_case("<test_case_id>")`
example: https://github.com/openstax/rex-web/blob/master/pytest-selenium/tests/test_toc.py#L5

The next marker we need to add tells the test framework that this test is "nondestructive". What this means is that this test is ok to run in production and that it doesn't do anything to manipulates data. If this marker is not here anytime the `base-url` matched on production it would skip this test.

`@markers.nondestructive`

Reply:
This is done with latest commit